reactioncommerce/reaction-next-starterkit
# [WIP] Feat 57 mikemurray segment pdp

Issue: #57
Impact: **minor**
Type: **feat**

## Issue

Add event tracking for `Product Viewed` event for the PDP page.

## Solution

- Add docs outlining data mappings
- Update `lib/tracking/trackProductViewed.js` to include outlined data mappings
- Ensure the `Product Viewed` is triggered for `variants` and `options`

## Testing

1. Create a segment account and get the API key
2. Start the app navigate to the PDP page
3. In the segment dashboard under `sources > javascript > debugger` see the event being tracked with proper data mappings
